draft-ietf-mls-protocol.md

```diff
@@ -3474,7 +3477,7 @@ The only proposal types defined in this document that do not require a path are:
 * `reinit`
 
 New proposal types MUST state whether they require a path. If any instance of a
-proposal type requires a path, then the proposal type requires a path. This
+proposal type requires a path, then the Commit requires a path. This
```

Comment:
```suggestion
proposal type requires a path, then the proposal type requires a path. This
```

Reverting because the emphasis here is on the "requires path" attribute being at the level of proposal types.